frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/ToggleListingCommitsAction.java

```diff
@@ -45,16 +44,6 @@ public void onUpdate(AnActionEvent anActionEvent) {
       return;
```

Comment:
whoops... according to PR #515 on line 40 should be getBranchLayoutWith**out**Logging

Reply:
Fixed... but it's super-hard to enforce, lemme solve that `systemically` on another PR :thinking:

Reply:
Ok, done on #534